### DISCUSSION TITLE
Any idea why QuickLook is unable to embed AV1?

### DISCUSSION THREAD

**Discussion by OneBigLotus** [QL-Win/QuickLook#1423] [Q&A]
I work in video archiving, and a lot of these files are AV1 encoded videos in .mp4 containers. QuickLook will play the audio, but it isn't able to view the video itself. Any plans to support this?

**Comment by FSXH1** [comment]
我也是打开AV1编码视频无法正常播放画面,也没找到合适的插件

**Comment by emako** [comment]
目前版本AV1应该是可以的
The current version should be OK for AV1
